File: apprefurbished/src/main/java/com/androidcollider/easyfin/database/DbHelper.java
package com.androidcollider.easyfin.database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;


public class DbHelper extends SQLiteOpenHelper {

    public static final String DATABASE_NAME = "FinU.db";
    private static final int DATABASE_VERSION = 1;


    public DbHelper(Context context) {
        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(SqlQueries.create_account_table);
        db.execSQL(SqlQueries.create_transactions_table);
        db.execSQL(SqlQueries.create_debt_table);
        db.execSQL(SqlQueries.create_rates_table);
    }

    // Method for update database
    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
    }

}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/repository/database/DbHelper.kt
package com.androidcollider.easyfin.common.repository.database

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

/**
 * @author Ihor Bilous
 */
class DbHelper(context: Context?) :
    SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(SqlQueries.create_account_table)
        db.execSQL(SqlQueries.create_transactions_table)
        db.execSQL(SqlQueries.create_debt_table)
        db.execSQL(SqlQueries.create_rates_table)
        db.execSQL(SqlQueries.create_transactions_category_expense_table)
        db.execSQL(SqlQueries.create_transactions_category_income_table)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        when (oldVersion) {
            1 -> {
                db.execSQL(SqlQueries.create_transactions_category_expense_table)
                db.execSQL(SqlQueries.create_transactions_category_income_table)
            }
        }
    }

    companion object {
        const val DATABASE_NAME = "FinU.db"
        private const val DATABASE_VERSION = 2
    }
}